src/main/java/com/epam/cdp/m2/hw2/aggregator/Java7Aggregator.java

```diff
@@ -1,23 +1,62 @@
 package com.epam.cdp.m2.hw2.aggregator;
 
-import java.util.List;
-
+import com.epam.cdp.m2.hw2.aggregator.comparator.StringLengthComparator;
 import javafx.util.Pair;
 
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.TreeSet;
+
 public class Java7Aggregator implements Aggregator {
 
-    @Override
-    public int sum(List<Integer> numbers) {
-        throw new UnsupportedOperationException();
+  @Override
+  public int sum(List<Integer> numbers) {
+    int sum = 0;
+    if (numbers != null) {
+      for (Integer number : numbers) {
+        sum += number != null ? number : 0;
+      }
     }
+    return sum;
+  }
 
-    @Override
-    public List<Pair<String, Long>> getMostFrequentWords(List<String> words, long limit) {
-        throw new UnsupportedOperationException();
+  @Override
+  public List<Pair<String, Long>> getMostFrequentWords(List<String> words, long limit) {
+    final TreeMap<String, Long> wordMap = new TreeMap<>();
+    final List<Pair<String, Long>> result = new ArrayList<>();
+    for (String word : words) {
```

Comment:
do not hesitate to extract out logical blocks into separate methods. That will make the code more cleaner and helps later to refactor and reuse it if needed.
Here are 2 logical sections based on 2 cycles. Good candidates for own private methods